R/data.R

```diff
@@ -613,6 +615,12 @@ spim_lancelot_data_rtm <- function(date, region, model_type, data, full_data,
 
   # ignore pillar 2 testing before 2020-06-18
   data[which(data$date < "2020-06-18"), cols_pillar2] <- NA_integer_
+  # after that date turn NAs to zeroes for columns where data is available
+  for (i in cols_pillar2) {
+    if (!all(is.na(data[, i]))) {
+      data[which(data$date >= "2020-06-18"), i] <- 0
+    }
+  }
```

Comment:
This is setting everything to 0 (and not just the NAs)

Reply:
I suggest it be `data[which(is.na(data[, i])), i] <- 0` and then move this above the "ignore pillar 2 testing before 2020-06-18" bit

Reply:
crap, sorry... mental fart... done